src/Microsoft.ServiceFabric.Client.Http/ImageStoreClient_.cs

```diff
@@ -37,6 +61,118 @@ internal partial class ImageStoreClient
                 cancellationToken);
         }
 
+        /// <inheritdoc />
+        public Task UploadFileChunkAsync(
+            byte[] fileChunkToUpload,
+            string pathInImageStore,
+            Guid? sessionId,
+            long startBytePosition,
+            long endBytePosition,
+            long length,
+            long? serverTimeout = 60,
+            CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return this.UploadFileChunkAsync(
+                fileChunkToUpload,
+                pathInImageStore,
+                sessionId,
+                endBytePosition,
+                endBytePosition,
+                length,
+                null,
+                serverTimeout,
+                cancellationToken);
+        }
+
+        /// <inheritdoc />
+        public async Task UploadApplicationPackageAsync(
+            string applicationPackagePath,
+            bool compressPackage = false,
+            string applicationPackagePathInImageStore = default(string),
+            CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (!Directory.Exists(applicationPackagePath))
+            {
+                throw new InvalidOperationException($"Application package path {applicationPackagePath} not found.");
+            }
+
+            var absPkgPath = FileUtilities.GetAbsolutePath(applicationPackagePath);
+            applicationPackagePath.ThrowIfNull(nameof(applicationPackagePath));
+
+            if (compressPackage)
+            {
+                await CompressApplicationPackage(absPkgPath);
+            }
+
+            var pkgPathInImageStore = applicationPackagePathInImageStore;
+            if (string.IsNullOrEmpty(pkgPathInImageStore))
+            {
+                pkgPathInImageStore = applicationPackagePath.Replace(Path.GetDirectoryName(applicationPackagePath), "");
+            }
+
+            pkgPathInImageStore = pkgPathInImageStore.Trim('\\', '/');
+
+            await LoadImageStoreConnectionString();
+            if (isLocalStore)
+            {
+                var files = Directory.EnumerateFiles(absPkgPath, "*", SearchOption.AllDirectories)
+                    .Select(file => new System.IO.FileInfo(file));
+
+                foreach (var file in files)
+                {
+                    var targetPath = Path.Combine(imageStorePath, Path.Combine(pkgPathInImageStore, file.FullName.Substring(absPkgPath.Length + 1)));
+                    if (!Directory.Exists(Path.GetDirectoryName(targetPath)))
```

Comment:
Add Braces for the block. 
I will add an editorconfig for these once generated code conforms to this as well.